WBooks/BaseViewController.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  BaseViewController.swift
+//  WBooks
+//
+//  Created by Megan Cole on 11/02/2020.
+//  Copyright © 2020 Wolox. All rights reserved.
+//
+
+import UIKit
+
+class BaseViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setNavigationBar()
+    }
+    
+    private func setNavigationBar() {
+        let navBar = navigationController!.navigationBar
```

Comment:
Let's use safe unwrapping:
```Swift
guard let navigationController = navigationController else { return }
```
(As you can see, it is allowed to define a constant with the same name as the optional you are unwrapping 😉)

Reply:
I renamed it with `navBar` to avoid using such a long variable name every time, since I have to use it many times.

Reply:
or are suggesting me to declare it like this?
```suggestion
        guard let navigationController = navigationController else { return }
        let navBar = navigationController.navigationBar```

Reply:
Yes, the last example is what I was mentioning 👍